app/src/main/java/fr/free/nrw/commons/utils/Constants.java

```diff
@@ -0,0 +1,9 @@
+package fr.free.nrw.commons.utils;
+
+public class Constants {
```

Comment:
Is there a need to create a new class for this?

Reply:
I prefer to keep all constants in  a single class to avoid kinds of confusion. I will check if there is already a common class like that. If not, can i not proceed with this class ?

Reply:
AFAIK we already use CommonsApplication.java for this purpose

Reply:
Perhaps you could put this constant with all the others first, and then create an issue for separating constants into another class, and submit another PR to transfer all of them if everyone is agreed? :) I think the most important part is that all constants are kept in the same place, so let's do that first, but I'm happy to have all constants moved to a separate class if that is the convention.